phpBB/styles/prosilver/template/viewtopic_body.html

```diff
@@ -302,7 +292,7 @@ <h3 {% if postrow.S_FIRST_ROW %}class="first"{% endif %}>
 			<!-- IF postrow.S_CAN_APPROVE -->
 			<form method="post" class="mcp_approve" action="{postrow.U_APPROVE_ACTION}">
 				<p class="post-notice unapproved">
-					<span><i class="icon fa-question icon-red fa-fw" aria-hidden="true"></i></span>
+					<span>{{ Icon('iconify', 'fa:question', '', true, 'c-reported-icon') }}</span>
```

Comment:
Is there still a benefit of having this wrapping `<span>`?

Reply:
its always good practice to wrap icon/title pairings